test/dlc_tests/ec2/tensorflow/inference/test_tensorflow_inference.py

```diff
@@ -278,15 +279,14 @@ def run_ec2_tensorflow_inference(
             )
         if telemetry_mode:
             check_telemetry(ec2_connection, container_name)
-    except:
-        inference_test_failed = True
+    except Exception:
         remote_out = ec2_connection.run(f"docker logs {container_name}", warn=True, hide=True)
         LOGGER.info(
             f"--- TF container logs ---\n--- STDOUT ---\n{remote_out.stdout}\n--- STDERR ---\n{remote_out.stderr}"
         )
+        raise
     finally:
         ec2_connection.run(f"docker rm -f {container_name}", warn=True, hide=True)
-    assert inference_test_failed is False, "tensorflow inference test failed"
```

Comment:
Removed this because it obfuscates the actual exception and traceback in the test logs. Instead, we can just raise the encountered exception after performing the required error handling.